src/inputs/labeled-inputs.scss

```diff
@@ -4,45 +4,36 @@
 @import '../icon/index';
 
 @mixin iui-input-container {
+  @include iui-reset;
   display: flex;
   flex-direction: column;
-  flex-wrap: nowrap;
   position: relative;
-  padding: 0;
-  margin: 0;
   text-align: left;
-  font-size: $iui-font-size;
   font-weight: $iui-font-weight-normal;
   cursor: default;
 
   // Pointer cursor used if this class is applied to a label element
   @at-root {
     label#{&} {
       cursor: pointer;
+    }
+  }
 
-      &.iui-disabled {
-        cursor: not-allowed;
-      }
+  &.iui-disabled {
+    &,
+    label {
+      cursor: not-allowed;
     }
   }
 
   > .iui-label {
-    padding: 0;
     margin-bottom: $iui-component-offset;
-    border: none;
-    display: block;
-    width: fit-content;
```

Comment:
Sure this is not needed? NOt label takes full width.. or that was bringin some issues somewhere?

Reply:
The whole component takes full width anyway, how does it change anything?

Reply:
I assumed, that rule was there for a reason. So now I am just commenting the removal of it.
We never look at the parent width, because at the end, everyones parent is body which takes it full.
It can have influence for random elements in that label, but idk...